tests/kernels/test_fused_quant_activation.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+import torch
+
+import vllm._custom_ops as ops
+from tests.kernels.utils import opcheck
+from vllm.model_executor.layers.activation import SiluAndMul
+
+DTYPES = [torch.bfloat16, torch.float16]
+QUANT_DTYPES = [torch.float8_e4m3fn]
+NUM_TOKENS = [1, 17, 86, 1234, 3045]  # Arbitrary values for testing
+HIDDEN_SIZES = [32, 64, 128, 2048, 4096]  # Arbitrary values for testing
```

Comment:
Could you add an odd hidden size if we support that? The nastiest problem size the kernel can handle

Reply:
The hidden size needs to be divisible by 2 so that the dimensions match up when we do the elementwise mat mul. I'll add some non-power-of-two sizes, though. We should also add an assert to the kernel that hidden_size % 2 == 0.